src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -3110,19 +3110,22 @@ public static int indexOf(final short[] array, final short valueToFind, int star
     }
 
     /**
-     * Returns whether a given array can safely be accessed at the given index.
+     * <p>Returns whether a given array can safely be accessed at the given index.
```

Comment:
Paragraph tags are not needed for the first paragraph of a Javadoc comment. When you do use HTML tags (elsewhere), you should close them.